src/firstscreen.cpp

```diff
@@ -28,7 +28,7 @@ FirstScreen::FirstScreen( PackageBackend *backend, QString *tmpFileName, const Q
     QVBoxLayout *mainLayout = new QVBoxLayout;
     mainLayout->setSpacing( 0 );
 
-    QFile dataFile( m_tmpFileName->toAscii() );
+    QFile dataFile( m_tmpFileName->toLatin1() );
```

Comment:
Note that we don't know what's the encoding used by the system of every user, so the right way to do it would be to use QString::toLocal8Bit, since that will use whatever the system is configured with, be it utf8 (usually), latin1 or whatever. Can you change that?